### DISCUSSION TITLE
Alternative for FormatterServices.GetUninitializedObject

### DISCUSSION THREAD

**Discussion by cecilphillip** [bchavez/Bogus#564] [Q&A]
I've used the solutions in mentioned in #334 for using records with bogus, which rely on `FormatterServices.GetUninitializedObject`. However `FormatterServices` is now marked as obsolete.

Does anyone have an alternative strategy they use to get around this?

**Comment by bchavez** [comment]
Hey @cecilphillip, hope you're doing well. Sorry about the late reply, been super busy recently.

I know lots of stuff in `FormatterServices` went obsolete, but as far as I know, I think the functionality is still present in `RuntimeHelpers`. 

For example, `RuntimeHelpers.GetUninitializedObject()`:

![image](https://github.com/user-attachments/assets/2a90cd2c-4915-4ab7-9bf0-fdfb2d2c21fc)

Let me know if that helps.

Thanks, Brian

**Reply by cecilphillip** [reply]
Hey @bchavez. Yeah it's been a long time. Hope things are good with you. 

Thanks for the response. I'll try that out.
